build.gradle

```diff
@@ -167,6 +167,17 @@ tasks.register('installLocalGitHookPreCommit', Copy) {
     }
 }
 
+tasks.register('runKeyboardProgram', Exec) {
+    workingDir = 'src/main/python'
+
+    commandLine 'python', 'keyboard_to_nt.py'
+}
+
+tasks.register('deployWithKeyboard') {
+    dependsOn("deploy")
+    dependsOn("runKeyboardProgram")
+    tasks.findByName('runKeyboardProgram').mustRunAfter('deploy')
```

Comment:
you can remove this line because you wrote the deploy dependency before the runKeyboardProgram dependency